app/views/one_payment.scala.html

```diff
@@ -0,0 +1,71 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.ViewConfig
+@import uk.gov.hmrc.play.views.html._
+@import model.des.BankDetails
+@import model.des.ObligationDates
+
+@this(
+mainTemplate: views.html.main_template
+)
+
+@(
+amountClaimed: String,
+obligationDates: ObligationDates,
+period: String,
+showBankDetails: Boolean,
+bankDetails: Option[BankDetails]
```

Comment:
see formatting advises in previous comments.

Reply:
Done